### PR TITLE
SLES-15-030730,030740,030750 corrections

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _Review required for SLES-15-030730,15-030740,15-030750_

#### Rationale:

- _Corrections according to requirement described here https://github.com/ComplianceAsCode/content/issues/7387#issuecomment-904429902_

- Fixes # _Issue number here (e.g. #26) or remove this line if no issue exists._

### CODE REVIEW

File: shared/templates/audit_rules_syscal_events/bash.template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[ "$(getconf LONG_BIT)" = "32" ] && RULE_ARCHS=("b32") || RULE_ARCHS=("b32" "b64")
+
+for ARCH in "${RULE_ARCHS[@]}"
+do
+	PATTERN="-a always,exit -F arch=$ARCH -S {{{ ATTR }}}.*"
+	GROUP="perm_mod"
+	FULL_RULE="-a always,exit -F arch=$ARCH -S {{{ ATTR }}} -F auid>={{{ auid }}} -F auid!=4294967295 -F key=perm_mod"
+
+	# Perform the remediation for both possible tools: 'auditctl' and 'augenrules'
+	fix_audit_syscall_rule "augenrules" "$PATTERN" "$GROUP" "$ARCH" "$FULL_RULE"
```
Comment (jan-cerny) [inline shared/templates/audit_rules_syscal_events/bash.template:17]: This bash function has been converted to a Jinja macro bash_fix_audit_syscall_rule. See its usage eg. in https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/auditing/auditd_configure_rules/audit_file_deletion_events/audit_rules_file_deletion_events/bash/shared.sh

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/rule.yml [hunk 2]
```diff
@@ -71,3 +90,9 @@ template:
           - renameat
           - unlink
           - unlinkat
+    {{% else %}}
+    name: audit_rules_syscal_events
```
Comment (jan-cerny) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/rule.yml:94]: It should be syscall (with 2 "l"s).

**Review by jan-cerny — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/rule.yml (hunk 2 above)
Comment (rumch-se) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/rule.yml]: Hello @jan-cerny and thank you very much for your proposal. I have done some corrections and this commit covers them https://github.com/ComplianceAsCode/content/pull/7529/commits/b8e1c44b3cc422d20b362097d72c49879049a2d8

**Comment by jan-cerny** [comment]
You have accidentally pushed `history.txt`, please remove it.

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**